prediction/tensorflow/README.md

```diff
@@ -7,11 +7,51 @@ TensorFlow trained model in AI Platform.
     `aiplatform-submit-train-job.sh` or `local-train.sh` scripts.
 *   Run `aiplatform-deploy-model.sh`
 
+## GPU support
```

Comment:
I wonder if it makes sense to also put a note at the top about needing access? (Since most people might not read to the bottom first?)